frontend/src/components/viewers/MetricsVisualizations.tsx

```diff
@@ -65,6 +68,8 @@ export function MetricsVisualizations({
     artifactTypes,
   );
   const verifiedMetricsArtifacts = getVerifiedMetricsArtifacts(artifacts, artifactTypes);
+  const verifiedHtmlArtifacts = getVertifiedHtmlArtifacts(linkedArtifacts, artifactTypes);
+  const verifiedMarkdownArtifacts = getVertifiedMarkdownArtifacts(linkedArtifacts, artifactTypes);
```

Comment:
nit: it's ok to skip in this PR

I think we can simplify the var names, because there are no unverifiedHtmlArtifacts in this scope, we can just call them htmlArtifacts, mdArtifacts etc to be concise.

Reply:
Sounds good, I have applied the change to shorten names.